workers/unity/Packages/com.improbable.gdk.querybasedinterest/Constraint.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+using UnityEngine;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    public class Constraint
+    {
+        public static ComponentInterest.QueryConstraint Sphere(double radius, Vector3 center)
+        {
+            return new ComponentInterest.QueryConstraint
```

Comment:
I'd have a method to return a pseudo "default(QueryConstraint)" (e.g. the empty lists set), use them here and then update the relevant fields e.g. 
```cs
var constraint = CreateDefault();
// constraint.SphereConstraint = ...;
```